doc/ref_model/chapters/appendix-a.md

```diff
@@ -1,28 +1,25 @@
 [<< Back](../../ref_model)
-# Appendix A - Guidelines For VNF Vendors
+# Appendix A - Guidelines
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [A.1 Goals](#A.1)
-* [A.2 Intro and Terminology](#A.2)
-* [A.3 VNF Maturity Levels](#A.3)
-* [A.4 Links](#A.3)
+* [A.1 Guidelines for VNF Vendors](#A.1)
+* [A.2 Guidelines for CNF Vendors](#A.2)
+* [A.3 VNF Transition Plan](#A.3)
+* [A.4 CNF Transition Phases](#A.4)
+* [A.5 Links](#A.5)
 
 <a name="A.1"></a>
-## A.1 Goals
-This Appendix has two goals:
-1. Provide guidance to VNF or more generally Application vendors on how to consume CNTT Reference Model and Architectures
-2. Provide usable definitions of maturity levels for VNF software architecture between Physical-to-Virtual migration and “Cloud Native”.
-
-The goal is not to be prescriptive on how to re-architect existing or architect new applications but rather staying within scope of focusing on interface and interaction between applications and platform.
+## A.1 Guidelines for VNF Vendors
 
 <a name="A.2"></a>
-## A.2 Intro and Terminology
-(Summary status and trends of ETSI NFV and Cloud. Decoupling application from platform. Terminology used/introduced.)
+## A.2 Guidelines for VNF Vendors
 
 <a name="A.3"></a>
-## A.3 VNF Maturity Levels
-(And how they map to CNTT RAs)
+## A.3 Guidelines for VNF Vendors
 
 <a name="A.4"></a>
 ## A.4 Links
```

Comment:
@rabi-abdel A.4 CNF VNF Transition Phases (or Plans)

Reply:
renamed to Generations.